pkg/restic/common_test.go

```diff
@@ -28,54 +28,106 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	velerov1api "github.com/heptio/velero/pkg/apis/velero/v1"
+	"github.com/heptio/velero/pkg/builder"
 	"github.com/heptio/velero/pkg/generated/clientset/versioned/fake"
 	informers "github.com/heptio/velero/pkg/generated/informers/externalversions"
 	velerotest "github.com/heptio/velero/pkg/util/test"
 )
 
-func TestGetPodSnapshotAnnotations(t *testing.T) {
+func TestGetVolumeBackupsForPod(t *testing.T) {
+	testPodName := "TestPod"
+	pvbtest1 := builder.ForPodVolumeBackup("velero", "pvb-1").Result()
```

Comment:
It would be more consistent with other tests to define the PVBs inline in the test cases (chaining together the builder methods like '.PodName(...).Volume(...)`, even if it means some duplication across test cases. I prefer this approach because then all the test data is right in front of you and you don't have to jump around to understand what data's being used.